src/app/products/manage-tax-components/create-tax-component/create-tax-component.component.ts

```diff
@@ -0,0 +1,148 @@
+/** Angular Imports */
+import { Component, OnInit } from '@angular/core';
+import { FormGroup, FormBuilder, Validators } from '@angular/forms';
+import { Router, ActivatedRoute } from '@angular/router';
+import { DatePipe } from '@angular/common';
+
+/** Custom Services */
+import { ProductsService } from '../../products.service';
+
+/**
+ * Create Tax Component component.
+ */
+@Component({
+  selector: 'mifosx-create-tax-component',
+  templateUrl: './create-tax-component.component.html',
+  styleUrls: ['./create-tax-component.component.scss']
+})
+export class CreateTaxComponentComponent implements OnInit {
+
+  /** Minimum start date allowed. */
+  minDate = new Date();
+  /** Maximum start date allowed. */
+  maxDate = new Date(2025, 6, 22);
```

Comment:
Any specific reason to keep the max date as this value? If no, please keep it the same as in other components (in general).

Reply:
That's the value I found in the current implementation in the community app [here](https://github.com/openMF/community-app/blob/develop/app/scripts/controllers/product/tax/CreateTaxComponentController.js)

Reply:
Okay let us update this, so it is not hardcoded:
```suggestion
  maxDate = new Date(new Date().setFullYear(new Date().getFullYear() + 10));
```